Formula/verilator.rb

```diff
@@ -26,22 +26,36 @@ class Verilator < Formula
   uses_from_macos "perl"
   uses_from_macos "python", since: :catalina
 
+  on_macos do
+    depends_on "gcc"
+  end
+
   skip_clean "bin" # Allows perl scripts to keep their executable flag
 
   # error: specialization of 'template<class _Tp> struct std::hash' in different namespace
   fails_with gcc: "5"
 
+  # Build hangs when using Clang
+  # https://github.com/Homebrew/homebrew-core/pull/124827
+  # https://github.com/Homebrew/homebrew-core/pull/130893
+  # https://github.com/Homebrew/homebrew-core/pull/133912
+  fails_with :clang
```

Comment:
Also had issue with LLVM Clang. Looks like this started since https://github.com/verilator/verilator/commit/93d50c44993cf879375a1ad93a1297a3fab0eaaf (previous commit didn't hang).